src/Collection.js

```diff
@@ -758,10 +757,6 @@ Collection.prototype.scroll = function (scrollId, options, filters, cb) {
 
   request.scrollId = scrollId;
 
-  if (options && options.scroll) {
```

Comment:
not needed here since `scroll` option is already managed within `kuzzle.query`method